src/test/java/org/kuali/kfs/module/ar/businessobject/TimeBasedBillingPeriodTest.java

```diff
@@ -1,89 +0,0 @@
-package org.kuali.kfs.module.ar.businessobject;
-
-import static org.junit.Assert.*;
-
-import java.sql.Date;
-import java.util.Calendar;
-
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.kuali.kfs.coa.service.AccountingPeriodService;
-import org.kuali.kfs.module.ar.ArConstants.BillingFrequencyValues;
-import org.mockito.Mockito;
-
-public class TimeBasedBillingPeriodTest {
```

Comment:
Are there any tests in here that would still be of value (once updated for compatibility), even after the consolidation of the billing period classes?

Reply:
Added test back and updated.